src/main.rs

```diff
@@ -166,10 +166,12 @@ fn dump_setting(path: &PathBuf) -> rekordcrate::Result<()> {
     Ok(())
 }
 
+#[cfg(feature = "xml")]
 fn dump_xml(path: &PathBuf) -> rekordcrate::Result<()> {
     let file = std::fs::File::open(path)?;
     let reader = std::io::BufReader::new(file);
-    let document: Document = quick_xml::de::from_reader(reader).expect("failed to deserialize XML");
+    let document: rekordcrate::xml::Document =
+        quick_xml::de::from_reader(reader).expect("failed to deserialize XML");
```

Comment:
have you figured out how to avoid the `expect` here yet? (the thing I criticized in the previous PR).